chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -292,16 +294,29 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
                     dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: event?.payload?.isChatHidden });
                 }
                 const dateNow = Date.now();
-                if (dateNow - lastLWICheckTimeRef.current > Constants.LWICheckOnVisibilityTimeout) {
-                    const conversationDetails = await getConversationDetailsCall(chatSDK);
+
+                if (window?.location?.href?.includes("open-in-window=true") && !window?.location?.href?.includes("is-popout-mode=true")) {
```

Comment:
A suggestion, can we move this to a util class and use function reference like 
isPopuOutChatMode

Reply:
done